components/base/sensorcontrolled/velocities.go

```diff
@@ -33,7 +33,7 @@ func (sb *sensorBase) setupControlLoop(linear, angular control.PIDConfig) error
 	// set the necessary options for a sensorcontrolled base
 	options := control.Options{
 		SensorFeedback2DVelocityControl: true,
-		LoopFrequency:                   20,
+		LoopFrequency:                   10,
```

Comment:
confirming this is lowered to match the wit imu frequency? has it caused worse behavior for SetVelocity?

Reply:
yeah I lowered this to match the wit. I can do some SetVelocity runs with odometry to confirm the behavior is the same

Reply:
thanks!

Reply:
following up on this, assuming there weren't any issues?

Reply:
there wasn't at the time, but rerunning again

Reply:
yeah no issues with SetVelocity + Odometry. The controller performed roughly the same at 10 Hz and 20 Hz based on the data I had. I did have a newish concern/note that I saw after doing these runs but they don't really matter for this PR